File: Domain/Dto/Business/GetBusinessDto.cs
using Domain.Entites;
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Dto.Business
{
    public class GetBusinessDto 
    {
        public Guid Id { get; set; }
        public string BusinessName { get; set; }
        public string BusinessDesc { get; set; }
        public string BusinussPhone { get; set; }
        public string MangerName { get; set; }    
        public string CityName { get; set; }
        public string AreaName { get; set; }
        public float BusinessRate { get; set; }

        public ICollection<string> Services{ get; set; }
        public string BusinessPic { get; set; }
    }
}

File: PetKeeperUI/src/app/_Models/GetBusinessDto.ts
import { Guid } from "guid-typescript";

export class GetBusinessDto {
    constructor(
        public  id : Guid,
        public  businessName: string,
        public  businessDesc: string,
        public  businussPhone: string,
        public  mangerName : string,
        public  cityName : string,
        public  areaName : string,
        public businessRate : number,
        public services : string[],
        public  businessPic : string,
    ) {
        this.id = Guid.create();
    }
}
